tests/unit/api/bucketPolicyAuth.js

```diff
@@ -377,6 +377,26 @@ describe('bucket policy authorization', () => {
             });
         });
 
+        it('should allow access to non-object owner for objectHead action with s3:GetObject permission',
+        function itFn(done) {
```

Comment:
why no arrow function like in the previous tests?

Reply:
It's to access the `this` variable, actually most of the previous tests have an actual function for that reason